src/components/Chart/index.d.ts

```diff
@@ -19,6 +20,8 @@ export interface ChartProps extends BaseProps {
     maintainAspectRatio?: boolean;
     plugins?: Array<object>;
     children?: ReactNode;
+    options?: ChartOptions;
+    [key: string]: unknown;
```

Comment:
I think here we only need to add options, the `[key: string]: unknown;` is only needed for dataset where you can pass extra props defined by plugins but here I don't think we need this